pkgs/applications/networking/instant-messengers/twinkle/default.nix

```diff
@@ -0,0 +1,66 @@
+{ stdenv,
+  fetchFromGitHub,
+  cmake,
+  libxml2,
+  libsndfile,
+  file,
+  readline,
+  bison,
+  flex,
+  ucommon,
+  ccrtp,
+  qt5,
+  alsaLib,
+  speex,
+  ilbc,
+  wrapQtAppsHook,
+  fetchurl,
+  bcg729 }:
+
+stdenv.mkDerivation rec {
+  pname = "twinkle";
+  version = "1.10.2";
+
+  src = fetchFromGitHub {
+    repo = pname;
+    owner = "LubosD";
+    rev = "v${version}";
+    sha256 = "0s0gi03xwvzp02ah4q6j33r9jx9nbayr6dxlg2ck9pwbay1nq1hx";
+  };
+
+  buildInputs = [
+    libxml2
+    file # libmagic
+    libsndfile
+    readline
+    ucommon
+    ccrtp
+    qt5.qtbase
+    qt5.qttools
+    qt5.qtquickcontrols2
```

Comment:
Please don't reference the `qt5` packageset per the qt packaging documentation 
* https://nixos.org/nixpkgs/manual/#sec-language-qt

and instead use `qtbase` attributes etc. formally